src/ptp.c

```diff
@@ -280,7 +280,7 @@ ptp_ioctl(struct tcb *const tcp, const unsigned int code,
 		CHECK_IOCTL_SIZE(PTP_SYS_OFFSET_PRECISE2, 64);
 
 		if (entering(tcp)) {
-			tprint_arg_next();
+			tprints_arg_next_name("argp");
 			return 0;
 		}
```

Comment:
Let's move this `tprints_arg_next_name("argp");` invocation after the `entering` branch.